README.md

```diff
@@ -6,36 +6,24 @@ By default, Split’s SDKs keep segment and split data synchronized as users nav
 
 [![Twitter Follow](https://img.shields.io/twitter/follow/splitsoftware.svg?style=social&label=Follow&maxAge=1529000)](https://twitter.com/intent/follow?screen_name=splitsoftware)
 
-## Compatibility
-Split Synchronizer supports Go version 1.8 or higher.
-
-## Getting started
-Below is a simple example that describes the instantiation of Split Synchronizer:
-
-### Usage via Go
-1. Install dependencies via `dep ensure`
-2. Then, execute `go run main.go -api-key "<YOUR_SDK_TYPE_API_KEY>" -redis-host "<YOUR_REDIS_HOST>" -redis-port <YOUR_REDIS_PORT> -redis-prefix "<YOUR_PREFIX>"`
-
-### Docker
-1. You can pull the Docker image from [Docker Hub](https://hub.docker.com/r/splitsoftware/split-synchronizer) and run it into your container environment.
+## Deploying to Heroku
+1. Create a Heroku app by going to the [private space](https://dashboard.heroku.com/teams/states-title/spaces) you want to add the new synchronizer to
```

Comment:
Looks you didn't finish the sentence

Reply:
Hmmm, the sentence is complete as is, but `create a Heroku app by going to the private space you want to add it to` could be rephrased to make it more clear. I'll update to something more like `Navigate to the private space you want to add the split synchronizer to, and create a Heroku app via the New->Create New App button`

Reply:
Updated with a `heroku create` equivalent that's less work and hopefully easier to follow